graylog2-server/src/main/java/org/graylog2/shared/journal/KafkaJournal.java

```diff
@@ -794,6 +803,31 @@ public Object apply(LogSegment segment) {
             return deletedSegments;
         }
 
+        /** Change the load balancer status from ALIVE to THROTTLE, or vice versa depending on the
+         * journal utilization percentage. As the utilization ratio is reliable only after cleanup,
+         * that's where this is called from.
+         */
+        private void updateLoadBalancerStatus(double utilizationPercentage) {
```

Comment:
I wondered about this. How about just using ServerStatus.publishLifecycle? Other parts of code seemed to use that.

With events, probably should check the status and send only actual changes. I would feel uneasy about EventBus unless cases of direct use would be removed. Mixing the two cases might cause unforeseen consequences.